gulpfile.js

```diff
@@ -49,6 +50,22 @@ gulp.task('watch', function () {
     gulp.watch('assets/css/**', ['css']);
 });
 
+gulp.task('build-dist', ['css'], function() {
+    var rimraf = require('rimraf');
+    var fs = require('fs');
+
+    var targetDir = 'dist/';
+    var themeName = JSON.parse(fs.readFileSync('./package.json')).name;
```

Comment:
This can be simplified to `themeName = require('./package.json').name;`

Reply:
Indeed, thanks!